### PR TITLE
Adding Dependency pkgs for ARM arch

### PR DESCRIPTION
**Pull request by Sabareesh-Kumar-Anandan**
Signed-off-by: Sabareesh Kumar Anandan <sanandan@marvell.com>

**- Why I did it**
Dependent packages (libssl-dev,libffi-dev) required for azure-storage

**- How I did it**
Added libssl-dev libffi-dev dependency packages

**- How to verify it**
Compiled for marvell-armhf

### CODE REVIEW

File: files/build_templates/sonic_debian_extension.j2 [hunk 1]
```diff
@@ -92,6 +92,12 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     python-yaml    \
     python-bitarray
 
+if [[ $CONFIGURED_ARCH == armhf || $CONFIGURED_ARCH == arm64 ]]; then
+    sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install \
+        libssl-dev \
+        libffi-dev
```
Comment (lguohan) [inline files/build_templates/sonic_debian_extension.j2:98]: why are these packages only installed for arm cpu?

Comment (Sabareesh-Kumar-Anandan) [reply files/build_templates/sonic_debian_extension.j2:98]: These packages are needed for installing azure-storage. We faced this issue while compiling for arm cpu.

Comment (lguohan) [reply files/build_templates/sonic_debian_extension.j2:98]: I do not see those packaged installed in x86 image.

```
test_rwuser@vlab-01:~$ dpkg --list | grep libssl
ii  libssl1.1:amd64                     1.1.1d-0+deb10u3            amd64        Secure Sockets Layer toolkit - shared libraries
test_rwuser@vlab-01:~$ dpkg --list | grep libffi
ii  libffi6:amd64                       3.2.1-9                     amd64        Foreign Function Interface library runtime
test_rwuser@vlab-01:~$ pip list | grep azure-sto
azure-storage                0.36.0   
```

Comment (Sabareesh-Kumar-Anandan) [reply files/build_templates/sonic_debian_extension.j2:98]: without these pkgs, we couldnt able to compile for arm. 
root@sonic:/home/admin# dpkg --list | grep libssl
ii  libssl1.1:armhf               1.1.0l-1~deb9u1                   armhf        Secure Sockets Layer toolkit - shared libraries
root@sonic:/home/admin# dpkg --list | grep libffi
ii  libffi6:armhf                 3.2.1-6                           armhf        Foreign Function Interface library runtime

Comment (lguohan) [reply files/build_templates/sonic_debian_extension.j2:98]: I am ok with this. can you move them around the azure-storage pip installation?

**Comment by Sabareesh-Kumar-Anandan** [comment]
This change is already merged via https://github.com/Azure/sonic-buildimage/pull/4639

**Pull request closed without merging**